### ISSUE TITLE
"No Channels" warning when selecting a alsa output of type multi

### ISSUE THREAD

**Issue by 1Yanik3** [sonosaurus/sonobus#127]
I've written some configuration for alsa in the .asoundrc file.
```
pcm.bluetooth {
        type plug
        slave {
            channels 2
            rate 48000
            pcm {
                type bluealsa
                device 94:DB:56:88:CE:85
                profile "a2dp"
            }
        }
        hint {
            show on
            description "XM4 Headsets"
        }
} 

pcm.both {
    type route;
    slave.pcm {
        type multi;

        slaves.a.pcm "bluetooth"
        slaves.a.channels 2;

        slaves.b.pcm "hw:CARD=Headphones,DEV=0"
        slaves.b.channels 2;

        bindings.0.slave a;
        bindings.0.channel 0;
        bindings.1.slave a;
        bindings.1.channel 1;
    
        bindings.2.slave b;
        bindings.2.channel 0;
        bindings.3.slave b;
        bindings.3.channel 1;
    }   
   
    ttable.0.0 1;
    ttable.1.1 1;
   
    ttable.0.2 1;  
    ttable.1.3 1;  
}


pcm.!default {
	type plug
	slave.pcm "both"
}

ctl.!default {
	type hw
	card SB
}
```

TLDR: what it does, is deliver the audio output to both my bluetooth headphones, as well as the headphone jack of the pi I'm using. This works fine in other applications.

However, SonoBus gives me a "Error when trying to open audio device! no channels" warning when I try to select it.

Any help would be greatly appreciated

**Comment by essej** [comment]
Did you fix the problem?
